extensions/tests/base_sql.go

```diff
@@ -0,0 +1,278 @@
+// Apache License 2.0
```

Comment:
I have refactored the test to https://github.com/xdblab/xdb/blob/main/persistence/sql/basetest to make it easier to maintain